ras_party/controllers/respondent_controller.py

```diff
@@ -74,7 +74,7 @@ def get_respondent_by_id(respondent_id, session):
 @with_db_session
 def delete_respondent_by_email(email, session):
     """
-    Delete a Respondent by its email
+    Delete a Respondent by its email fter disabling all their enrolments
```

Comment:
Spelling mistake

Reply:
Also, this isn't true.  This function doesn't care about the state of enrolments.  All this function does is delete a respondent.

Reply:
Missed it after reverting change , good catch